src_py/apiServer/apiServer.py

```diff
@@ -43,13 +58,21 @@ def __init__(self):
         self.transmitter = Transmitter(self.mainServerAddress)
 
         print("\n***Please remember to execute NerlnetRun.sh before continuing.")
-        
+
     def sendJsonsToDevices(self):
         # Send the content of jsonPath to each devices:
         print("\nSending JSON paths to devices...")
 
-        archAddress = globe.content[0][:-1]
-        connMapAddress = globe.content[1][:-1]
+        # archAddress = globe.content[0][:-1]
+        # connMapAddress = globe.content[1][:-1]
+
+        archAddress , connMapAddress, exp_flow_json = self.getUserJsons()
+        [bad, archPath] = archAddress.split("/NErlNet")
```

Comment:
What is bad in here? 
bad is a bad name for a variable 😃 please change it to a more informative name. 
There is also a hidden bug here. If  "/NErlNet" appears more than once in path?

Reply:
Var named changed to JsonsPath.
Jsons are always saved in "/NErlNet/inputJsonFiles/{JSON_TYPE}/file.json", so there isn't another NErlNet dir on that path.